romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
@debedb ideally we should fix `/romana` at some point but not in this thread. For now, it would be nice to avoid complicating the matter further by putting more unrelated stuff into it.